src/lib/components/dateSelector/index.tsx

```diff
@@ -63,41 +63,49 @@ const DateSelector = ({
   const availableMonths = moment.monthsShort();
 
   const [date, setDate] = useState<Partial<CalendarDate>>(
-    CalendarDateValue ?? {}
+    CalendarDateValue ?? {},
```

Comment:
It look like it shouldn't be starting with a capital letter (these are only for react components). I know you're not the one that made this change but can you update it so it's not so confusing?